ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -134,6 +135,11 @@
   {:default 3600
    :type Long})
 
+(defconfig bulk-update-status-table-cleanup-interval
+  "Number of seconds between cleanup of the old status rows."
+  {:default 86400 
```

Comment:
Can you put a comment after 86400 to make that number human readable `86400 ;; 24 hours`?